packages/account/src/Styles/account.scss

```diff
@@ -1573,7 +1568,7 @@ $MIN_HEIGHT_FLOATING: calc(
         &-field {
             margin-bottom: 3.2rem;
 
-            @include mobile {
+            @include mobile-or-tablet-screen {
```

Comment:
hey, do the files inside `account-common.scss` also require some change? Can we crosscheck if anything else needs to be modified there. 🙏

Reply:
Hi, I can check but as far as I saw in UI everything looks fine to me 😁